mappings/net/minecraft/world/entity/ai/memory/NearestVisibleLivingEntities.mapping

```diff
@@ -0,0 +1,10 @@
+CLASS net/minecraft/unmapped/C_bfzvfpbw net/minecraft/world/entity/ai/memory/NearestVisibleLivingEntities
```

Comment:
`world/entity` is not a package that QM uses. 
```suggestion
CLASS net/minecraft/unmapped/C_bfzvfpbw net/minecraft/entity/ai/memory/NearestVisibleLivingEntities
```
This will require moving the file too.